### DISCUSSION TITLE
Build oboe library without Android Studio

### DISCUSSION THREAD

**Discussion by Mario100000** [google/oboe#1568] [Q&A]
I have an app which uses cmake.

If I wanted to use oboe library I would need oboe static library (something as liboboe.a arm64) and headers.

Your guide does not seem to consider this scenario.

I tried "cmake ." but it loops with error:

> CMake Error at CMakeLists.txt:69 (add_library):
>   add_library cannot create target "oboe" because another target with the
>   same name already exists.  The existing target is a static library created
>   in source directory "/home/due/Localcp/working/oboe-main".  See
>   documentation for policy CMP0002 for more details.
> 
> 
> CMake Error at CMakeLists.txt:93 (target_link_libraries):
>   Attempt to add link library "log" to target "oboe" which is not built in
>   this directory.
> 
>   This is allowed only when policy CMP0079 is set to NEW.

Is there a procedure to build library standalone (I'm using debian)?

**Comment by grybouilli** [comment]
any news on this ? @Mario100000 have you ever managed to build the lib for this type of purpose ?
